2-mandatory/8-password-validator.js

```diff
@@ -23,7 +23,19 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {}
+function validatePasswords(passwords) {
+  let pass = passwords.length >= 5;
+  let number = containsNumber(passwords);
+  let regex = containsSymbol(passwords);
+  let lowercase = containsLowercaseLetter(passwords);
+  let uppercase = containsUppercaseLetter(passwords);
+
+  if (pass && number && regex && lowercase && uppercase) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
Not quite right. The first thing to notice here is that the functions take an array of passwords as parameters and should be returning an array of boolean.